blueprints/addon/files/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    // The combination of `baseUrl` with `paths` allows Ember's classic package
+    // layout, which is not resolvable with the Node resolution algorithm, to
+    // work with TypeScript.
+    "baseUrl": ".",
+    "paths": {
+      "dummy/tests/*": ["tests/*"],
+      "dummy/*": ["tests/dummy/app/*", "app/*"],
+      "my-addon": ["addon"],
```

Comment:
could `my-addon` be substituted with the `package.json#name`
Not everyone will use `my-addon` 😅

Reply:
Oh geez! 🙈 😂

But tests are passing at least... 🙃

Reply:
Fixed!